tests/integrationtests/tst_perfparser.cpp

```diff
@@ -743,8 +837,8 @@ private slots:
 
         VERIFY_OR_THROW(recordingFinishedSpy.wait(10000));
 
-        COMPARE_OR_THROW(recordingFailedSpy.count(), 0);
```

Comment:
this and the changes below come from the first/second review from @milianw; I've adjusted this in both the new and the old tests (which were used as a template for the original ones)

Reply:
great, thanks - but please make this a separate commit - first do the cleanup, then add the new work in a second patch on top